marimo/_plugins/ui/_impl/dataframes/dataframe.py

```diff
@@ -82,13 +82,15 @@ class dataframe(UIElement[Dict[str, Any], "pd.DataFrame"]):
     **Initialization Args.**
 
     - `df`: the DataFrame or series to transform
+    - `page_size`: the number of rows to show in the table
     """
 
     _name: Final[str] = "marimo-dataframe"
 
     def __init__(
         self,
         df: pd.DataFrame,
+        page_size: Optional[int] = 5,
```

Comment:
Placing this here instead of the end is technically a breaking change - we should place this after the on_change